Controller/Adminhtml/Menu/MassDelete.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Snowdog\Menu\Controller\Adminhtml\Menu;
+
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Ui\Component\MassAction\Filter;
+use Snowdog\Menu\Model\ResourceModel\Menu\CollectionFactory;
+use Snowdog\Menu\Api\MenuRepositoryInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Exception;
+
+/**
+ * Class MassDelete
+ *
+ * This action deletes multiple menus
+ */
+class MassDelete extends Action implements HttpPostActionInterface
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'Snowdog_Menu::menus';
+
+    /**
+     * @var Filter
+     */
+    protected $filter;
```

Comment:
Is there any reason why these properties are set as `protected`, it looks like they are not going to be overriden/used in a child class, I think we could switch them to `private` instead.

Reply:
There are other properties that are `protected`, that is why I left it also as `protected`. Now I changed it to `private`